src/settings.rs

```diff
@@ -191,16 +165,6 @@ fn present_message(heading: &str, body: &str, transient_for: Option<&impl IsA<gt
     dialog.present();
 }
 
-fn is_accessible(path: &Path) -> bool {
```

Comment:
Why was this removed?

Reply:
The file chooser now gives access to the selected folder when sandboxed

Reply:
The problem I had before is the portal returns path that starts with `/run` when the path is only accessible through the portal. Since we save the path to the settings, it is no longer accesible too once we relaunch the application, or restart the portal process.

I'm not sure if thus bug still exists, though.

Reply:
It didn't seem to be the case when I had tested by both closing the app and restarting Fedora entirely

Reply:
As far as I remember, It will only be reproducible when you have the app installed, instead of running from builder or vscode.

You also have to select folders that Kooha doesn't have permission to like Documents folder

Reply:
Yes I had the flatpak exported and then installed

Reply:
It now works too on my side, it probably has been fixed on the portal side